.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please add .vscode folder to gitignore
Refrerence: 
https://stackoverflow.com/questions/7527982/applying-gitignore-to-committed-files/7528016